backend/libexecution/serialization_converters.ml

```diff
@@ -180,6 +180,8 @@ and fromFluidExpr (expr : Libshared.FluidExpression.t) :
         Filled (id, FluidLeftPartial (str, fromFluidExpr ~inPipe oldVal))
     | EList (id, exprs) ->
         Filled (id, ListLiteral (List.map ~f:r exprs))
+    | ETuple (id, exprs) ->
```

Comment:
I added the `TupleLiteral` but it wasn't compiling when I added it to last place in the `enum`. It is working when I put it right after the `ListLiteral`.

Reply:
It definitely has to go in last place. Otherwise it breaks the binary serialization format we use to store code in the DB (see https://github.com/darklang/dark/blob/main/docs/oplist-serialization.md)